question/bank/previewquestion/amd/src/preview.js

```diff
@@ -16,16 +16,102 @@
 /**
  * Javascript for preview.
  *
+ * @module     qbank_preview/preview
  * @copyright  2021 Catalyst IT Australia Pty Ltd
  * @author     Safat Shahin <safatshahin@catalyst-au.net>
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
+import $ from 'jquery';
 
+/**
+ * Set up the actions.
+ *
+ * @method init
+ * @param {bool} redirect Redirect.
+ */
 export const init = (redirect) => {
     if (!redirect) {
         let closeButton = document.getElementById('close-previewquestion-page');
         closeButton.onclick = () => {
             window.close();
         };
     }
+
+    // Set up the form to be displayed.
+    setupQuestionForm('responseform');
+};
+
+/**
+ * Set up the form element to be displayed.
+ *
+ * @method setupQuestionForm
+ * @param {string} formElement The form element.
+ */
+const setupQuestionForm = (formElement) => {
+    let form = document.getElementById(formElement);
```

Comment:
i think the formelement name should be here directly, it's an identifier from the mustache of the preview plugin, and if mod_quiz wants to call this, it should just call this instead of passing the id from the call, same goes for other plugins.